commands/run.sh

```diff
@@ -241,15 +241,35 @@ fi
 
 # Start up service dependencies in a different header to keep the main run with less noise
 if [[ "$(plugin_read_config DEPENDENCIES "true")" == "true" ]] ; then
+  exitcode=0
+
   echo "~~~ :docker: Starting dependencies"
   if [[ ${#up_params[@]} -gt 0 ]] ; then
-    run_docker_compose "${up_params[@]}" up -d --scale "${run_service}=0" "${run_service}"
+    if ! run_docker_compose "${up_params[@]}" up -d --scale "${run_service}=0" "${run_service}"; then
+      exitcode=1
+    fi
   else
-    run_docker_compose up -d --scale "${run_service}=0" "${run_service}"
+    if ! run_docker_compose up -d --scale "${run_service}=0" "${run_service}"; then
+      exitcode=1
+    fi
   fi
```

Comment:
I don't believe that the proposed code will preserve the exit code(I think `$?` will be `0`) but I'll check that and put up a change to preserve the exit code shortly.

Reply:
I may need to stop doing code review past midnight then :P

If you are correct it may be a better idea to do something like `run_docker_compose ... || exitcode=$?`

Reply:
Just checked and `$?` is `0` when you use it inside `if ! false; then`. Using `|| exitcode=$?` is much clearer than the method I had in mind so I've switched to using it.